ps2/inttypes.h

```diff
@@ -0,0 +1,23 @@
+/* RetroArch - A frontend for libretro.
+ * Copyright (C) 2010-2018 - Francisco Javier Trujillo Mata - fjtrujy
+ *
+ * RetroArch is free software: you can redistribute it and/or modify it under the terms
```

Comment:
Does this header need to be GPL? It's tiny and taints the license of this repo.

Reply:
Up to us, thus class was generated/created by me. So whatever you may think that we need to put there it.
This file is just copied from RetroArch project

Reply:
Since it's just definitions and no actual code just matching the MPL license seems like a good idea (even libretro.c does that, though I wrote that.)

Reply:
@endrift This is just for getting it to compile on PS2, since it lacks even inttypes.h. They are just headers from RetroArch/libretro-common and it's frankly innocuous. I don't see the big deal.

Reply:
As for myself, I don't really want to make a big deal over this - I would not mind if you insist on the license header being removed, if @fjtrujy is OK with that for this repo and if that resolves this issue for you and we can merge the PR as is, then I'd be in favor of going for that.

Reply:
I would greatly appreciate that, and it would reduce my issues with this PR to the 555/565 issue, which mostly seems like an issue with conflicting semantics over the ifdefs.

Reply:
OK, he will push the commits for this tomorrow.

Reply:
Removed